install_config/certificate_customization.adoc

```diff
@@ -18,6 +18,24 @@ xref:../architecture/infrastructure_components/web_console.adoc#architecture-inf
 This can be done during a
 xref:../install/configuring_inventory_file.adoc#advanced-install-custom-certificates[cluster installation] or configured after installation.
 
+[[using-certificate-chains]]
+== Using Certificate Chains
```

Comment:
Would a user think of using a certificate chain or multiple certificates? I'm also always in favor of active, "you" centered statements instead of passive statements.

Reply:
I replaced this title with `Configuring a Certificate Chain`. I think the verbiage of `certificate chain` is common enough that it'll make sense to people who need this section.